app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        issues_found = 0
```

Comment:
The `issues_found` variable is incremented, but since an exception is raised immediately after, it will never be greater than 1. Consider removing `issues_found` or using it differently if you intend to collect multiple issues before raising an exception.